Description: Remove use of __DATE__ to make the build reproducible
Author: James Cowgill <jcowgill@debian.org>
Forwarded: no
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/libpcsxcore/r3000a.c
+++ b/libpcsxcore/r3000a.c
@@ -31,7 +31,7 @@ R3000Acpu *psxCpu = NULL;
 psxRegisters psxRegs;
 
 int psxInit() {
-	SysPrintf(_("Running PCSXR Version %s (%s).\n"), PACKAGE_VERSION, __DATE__);
+	SysPrintf(_("Running PCSXR Version %s.\n"), PACKAGE_VERSION);
 
 #ifdef PSXREC
 	if (Config.Cpu == CPU_INTERPRETER) {
